web/src/billing/deactivate_server.ts

```diff
@@ -3,7 +3,7 @@ import $ from "jquery";
 export function initialize(): void {
     $("#server-deactivate-form").validate({
         submitHandler(form) {
-            $("#server-deactivate-form").find(".loader").css("display", "inline-block");
+            $("#server-deactivate-form").find(".loader").show();
```

Comment:
@PedroSimas4 have you tested the changes manually? I haven't dug deep into the PR at the moment, but from an initial look, the equivalent of `show()` in JQuery is `.css( "display", "block" )`, but we want an `inline-block` over here. This logic would work when the element already has a `display` property applied on it, and we hide it using `hide()` where the property is saved in jQuery's data cache, to be retrieved back when using `show()`.